common/Geometry2d/Point.hpp

```diff
@@ -258,16 +258,17 @@ class Point {
 
     /**
     * Returns a vector with the same direction as this vector but with magnitude
-    * one.
+    * given,
+    * unless this vector is zero.
     * If the vector is (0,0), Point(0,0) is returned
     */
-    Point normalized() const {
+    Point normalized(float magnitude = 1.0) const {
```

Comment:
I'd kinda prefer if we kept this the way it was before.  The definition of "normalized" is that it's a unit vector.  You can achieve the same thing with `myVector.normalized() * magnitude`.